drivers/sensor/ti_hdc/ti_hdc.c

```diff
@@ -123,12 +131,14 @@ static int ti_hdc_init(struct device *dev)
 		LOG_ERR("Failed to get correct manufacturer ID");
 		return -EINVAL;
 	}
-	if (read16(drv_data->i2c, DT_TI_HDC_0_BASE_ADDRESS,
-		   TI_HDC_REG_DEVICEID) != TI_HDC_DEVICEID) {
-		LOG_ERR("Failed to get correct device ID");
+	tmp = read16(drv_data->i2c, DT_TI_HDC_0_BASE_ADDRESS,
+		     TI_HDC_REG_DEVICEID);
+	if (tmp != TI_HDC1000_DEVID && tmp != TI_HDC1050_DEVID) {
+		LOG_ERR("Unsupported device ID");
```

Comment:
Sorry for the nitpick, but the changes here and in `ti_hdc_channel_get` are unrelated to the commit message `sensors: ti_hdc driver wait conversion support` and should be moved to a separate commit

Reply:
done